.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  NET_Tests:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        dotnet-version: [ '35', '40' ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup variables
+        id: test_file
+        run: echo "::set-output name=test_file::EasyPost.Tests.Net${{ matrix.dotnet-version }}"
+      - name: Setup MSBuild
+        if: always()
+        uses: microsoft/setup-msbuild@v1.1
+      - name: Setup Nuget
+        uses: NuGet/setup-nuget@v1.0.5
+      - name: Setup VSTest
+        uses: darenm/Setup-VSTest@v1
+      - name: Navigate to project root
```

Comment:
GitHub workflows should already be at the project root, AKA the `GITHUB_WORKSPACE` by default? Is there a reason we are changing directories here?

Reply:
Was simply following the tutorial I found, but you're right, I don't think we ever leave, so this step might be unnecessary.